Preference Access SSN Matching/REPWRITERSPECS/RB.PREFACCESS.SSNMATCHING

```diff
@@ -0,0 +1,1628 @@
+     ELSE
+      DO
+       BNOTEXT=BNOCFGFILELINE
+       CALL BNONLTS
+       BNOCFGFILELINE=BNOTEXT
+       BNOPOS=CHARACTERSEARCH(BNOCFGFILELINE,":")
+       IF SEGMENT(BNOCFGFILELINE,1,1)<>"*" THEN [Check for comment line]
+        DO
+         IF BNOPOS>0 THEN [line is a parameter setting]
+          DO
+           IF BNOCFGPARAMCOUNT<BNOCFGPARAMMAX THEN
+            DO
+
+             BNOCFGPARAMCOUNT=BNOCFGPARAMCOUNT+1
+             BNOPARAMTMP=SEGMENT(BNOCFGFILELINE,1,BNOPOS-1)
+             BNOTEXT=SEGMENT(BNOCFGFILELINE,BNOPOS+1,LENGTH(BNOCFGFILELINE))
+             CALL BNONLS
+             BNOPARAMVALTMP=BNOTEXT
+             IF BNOCFGPARAMCOUNT>0 THEN
+              DO
+               BNOPARAMFOUND=FALSE
+               FOR BNOTMPLOOP=1 TO BNOCFGPARAMCOUNT
+                DO
+                 IF BNOCFGPARAM(BNOTMPLOOP)=BNOPARAMTMP THEN
+                  BNOPARAMFOUND=TRUE
+                END
+               IF BNOPARAMFOUND=TRUE THEN
+                DO
+                 BNOERRORCODE=BNOECODECFGDUPLICATEENTRY
+                END
+               ELSE
+                DO
+                 BNOCFGPARAM(BNOCFGPARAMCOUNT)=BNOPARAMTMP
+                 BNOCFGPARAMVAL(BNOCFGPARAMCOUNT)=BNOPARAMVALTMP
+
+[* Since the program is in the SETUP division, print out the values
+** of any parameter settings to the batch output report
+*]
+                 PRINT FORMAT("99: ",BNOCFGPARAMCOUNT)+BNOCFGPARAM(BNOCFGPARAMCOUNT)+"-"+
+                       BNOCFGPARAMVAL(BNOCFGPARAMCOUNT)
+                 NEWLINE
+                END
+              END  [IF BNOCFGPARAMCOUNT>0]
+            END  [IF BNOCFGPARAMCOUNT<BNOCFGPARAMMAX]
+          END
+        END [IF SEGMENT(BNOCFGFILELINE,1,1)<>"*"]
+      END  [IF BNOCFGFILEFERROR=""]
+    END  [WHILE BNOCFGFILEFERROR=""]
+  END
+ FILECLOSE(BNOCFGFILENUMBER,BNOCFGFILEFERROR)
+END [PROCEDURE]
+
+PROCEDURE BNOSETPARAMVALS
+[* Retrieve parameter settings
+**
+*]
+ FOR BNOLOCALLOOP=0 TO BNOACCTTYPESMAX
+  DO
+   PARAMINVHOMEACCTTYPES(BNOLOCALLOOP)=FALSE
+   PARAMINVTARGETACCTTYPES(BNOLOCALLOOP)=FALSE
+  END
+ FOR BNOLOCALLOOP=0 TO BNOSLTYPESMAX
+  DO
+   PARAMINVTARGETSHARETYPES(BNOLOCALLOOP)=FALSE
+   PARAMINVTARGETLOANTYPES(BNOLOCALLOOP)=FALSE
+  END
+ FOR BNOLOCALLOOP=1 TO BNOEXTLOANTYPESMAX
+  DO
+   PARAMINVTARGETEXTLOANTYPES(BNOLOCALLOOP)=FALSE
+  END
+ FOR BNOLOCALLOOP=0 TO BNONAMETYPESMAX
+  DO
+   PARAMVALIDNAMETYPES(BNOLOCALLOOP)=FALSE
+  END
+
+ PARAMEXCLUDEIRSCODES=TRUE
+ PARAMSETDPFLAG=FALSE
+ PARAMSETWDFLAG=FALSE
+ PARAMSETIQFLAG=FALSE
+ PARAMPROCESSPAFLAG=PAVALIDATE
+ BNODEBUGMODE=FALSE
+ TESTACCOUNTS=""
+ PARAMINVALIDSSN=""
+ MAXACCOUNTS=0
+
+ BNOPARAMFIND="IHAT"
+ CALL BNOGETPARAMVAL
+ IF BNOPARAMFOUND=TRUE THEN
+  DO
+   LELISTINPUT=LOWERCASE(BNOPARAMVAL)
+   IF LELISTINPUT="" THEN
+    LELISTINPUT="none"
+   CALL LISTEXPAND
+   FOR BNOLOCALLOOP=0 TO BNOACCTTYPESMAX
+    DO
+     PARAMINVHOMEACCTTYPES(BNOLOCALLOOP)=LELIST(BNOLOCALLOOP)
+    END
+   BNODEBUGLFLINE="IHAT:"+BNOPARAMVAL
```

Comment:
I don't think you need this because you are printing if Debug is true already in Letter File PREFACCESS.SSNMATCHING.DB